src/main/java/com/exactpro/th2/common/schema/grpc/router/GrpcRouter.java

```diff
@@ -39,7 +41,17 @@
      * @param cls service class
      * @return service
      */
-    <T> T getService(@NotNull Class<T> cls) throws ClassNotFoundException;
+    <T> T getService(@NotNull Class<T> cls);
+
+    /**
+     * Returns the gRPC service for each endpoint specified for passed class
+     * @param <T> the type of gRPC service to create
+     * @param serviceClass the class of corresponding service
+     * @return the service for each endpoint specified in the configuration
+     */
+    default <T> Set<T> getServices(@NotNull Class<T> serviceClass) {
```

Comment:
Why do we use Set instead of List?

Reply:
Because there should not be any dependencies on the order of elements. Also, it shows that each service is unique.

Reply:
We should hint the developer about the requirement to implement equal / hashCode methods in class T to guarantee uniqueness.

Reply:
Should we? We generate those implementations by proto files. Otherwise, they won't work like common expects.
But if we are afraid that the developer will write a new implementation with unpredictable equal / hashCode I will change it to a List